Metro Skin Installer/Metro Skin Installer/MainForm.cs

```diff
@@ -20,20 +16,21 @@ public MainForm()
         {
             InitializeComponent();
             CheckForIllegalCrossThreadCalls = false;
-            if (!hasPerimssion(FindSteamSkinDir()))
+            if (!hasPermission(FindSteamSkinDir()))
```

Comment:
If Steam directory isn't found in registry, it will try to look for permission in C:\, and will display it doesn't have permission